index.html

```diff
@@ -46,30 +48,256 @@ <h2 id="jobTitle">I am a Software Developer</h2>
+            <div class="project-link">
+                <p class="html">html</p>
+                <p class="bootstrap">bootstrap</p>
+                <p class="ruby">Ruby</p>
+            </div>
+            <p class="goto_project">See Project</p>
+        </div>
+
+        <div class="content_1 grid-content"><img src="images\img_4.png" alt="">
+            <h3 class="title-post">Profesional Art Printing Data More</h3>
+            <p class="supporting-text">A daily selection of privately personalized
+                reads; no accounts or sign-ups required.
+                has been the industry's standard</p>
+            <div class="project-link">
+                <p class="html">html</p>
+                <p class="bootstrap">bootstrap</p>
+                <p class="ruby">Ruby</p>
+            </div>
+            <p class="goto_project">See Project</p>
+        </div>
+
+        <div class="content_1 grid-content"><img src="images\img_4.png" alt="">
+            <h3 class="title-post">Profesional Art Printing Data More</h3>
+            <p class="supporting-text">A daily selection of privately personalized
+                reads; no accounts or sign-ups required.
+                has been the industry's standard</p>
+            <div class="project-link">
+                <p class="html">html</p>
+                <p class="bootstrap">bootstrap</p>
+                <p class="ruby">Ruby</p>
+            </div>
+            <p class="goto_project">See Project</p>
+        </div>
+
+        <div class="content_1 grid-content"><img src="images\img_4.png" alt="">
+            <h3 class="title-post">Profesional Art Printing Data More</h3>
+            <p class="supporting-text">A daily selection of privately personalized
+                reads; no accounts or sign-ups required.
+                has been the industry's standard</p>
+            <div class="project-link">
+                <p class="html">html</p>
+                <p class="bootstrap">bootstrap</p>
+                <p class="ruby">Ruby</p>
+            </div>
+            <p class="goto_project">See Project</p>
+        </div>
+
+        <div class="content_1 grid-content"><img src="images\img_4.png" alt="">
+            <h3 class="title-post">Profesional Art Printing Data More</h3>
+            <p class="supporting-text">A daily selection of privately personalized
+                reads; no accounts or sign-ups required.
+                has been the industry's standard</p>
+            <div class="project-link">
+                <p class="html">html</p>
+                <p class="bootstrap">bootstrap</p>
+                <p class="ruby">Ruby</p>
+            </div>
+            <p class="goto_project">See Project</p>
+        </div>
+
+    </section>
+</section>
+
+<!-- About-me Section -->
+
+<section class="about_me_grid">  
+    <div class="about_me"> 
+  <svg id="about_img" width="586" height="354" viewBox="0 0 586 354" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
- [ ] Even though you did a really good job applying this SVG to make it follow the original designs, it is not supposed to be used as an SVG. Instead, you have to use a CSS background image in order to achieve this, so please apply the necessary changes here.